core/templates/dev/head/css/oppia.css

```diff
@@ -112,10 +112,13 @@ body {
   margin-bottom: 30px;
 }
 
-.oppia-profile-top-card {
+.oppia-profile-user-card {
   background-color: #fff;
-  margin: 0 0 24px 0;
-  padding: 24px;
+  width: 220px;
```

Comment:
Here and below: alphabetize CSS attributes for each rule. (This makes it easier to scan them to find the value of a property.)